packages/website/src/features/Packages/FunctionInput/TupleInput.tsx

```diff
@@ -0,0 +1,70 @@
+import { Flex, FormControl, FormLabel, Text } from '@chakra-ui/react';
+import { FunctionInput } from '../FunctionInput';
+import { useState } from 'react';
+import { BigNumber } from 'ethers';
+import { AbiParameter } from 'viem';
+
+const TupleInput = ({
+  input,
+  handleUpdate,
+}: {
+  input: any;
```

Comment:
Can this be typed as `input: AbiParameter;` ?

Reply:
Good Catch! sure! Thanks @mjlescano